src/panels/OriginPanel/AnchorAimView/AnchorAimView.tsx

```diff
@@ -151,7 +152,7 @@ export function AnchorAimView({ favorites, searchableNodes, matcherFunction }: P
         )}
         matcherFunc={matcherFunction}
       >
-        <FilterList.SearchResults.VirtualList />
+        <FilterList.SearchResults.VirtualList gap={3} />
```

Comment:
I'm a bit sad about this custom gap in both the focus view and here. As well as the added `mb`

3 is a new number 🥲 how would it look with 2?